### PR TITLE
added lwa buffer triggering

### PR DESCRIPTION
**Pull request by nkosogor**
@jack-h can you take a look at the code with buffer triggering in buffer_trigger.py, which I wrote based on one of our zooms?

### CODE REVIEW

File: ovro_alert/buffer_trigger.py
Comment (caseyjlaw) [inline ovro_alert/buffer_trigger.py:1]: I think it would be ok to have this script be in a block of code at the bottom or `lwa_alert_client.py`. You would just need to have a line like:
`if __name__ == '__main__':`

and follow it with the contents of this file. That would make it easier to maintain the OVRO-LWA code.

File: ovro_alert/buffer_trigger.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from time import sleep
+from astropy.time import Time
+from ovro_alert.lwa_alert_client import LWAAlertClient
+from mnc import control
+
+
+xhosts = ['lxdlwagpu02']
+con = control.Controller('proj/lwa-shell/mnc_python/config/lwa_config_calim.yaml', xhosts=xhosts)
+pipelines = con.pipelines
+p = pipelines[0]
```
Comment (caseyjlaw) [inline ovro_alert/buffer_trigger.py:10]: Do we actually need to send a trigger command to all pipelines? And each would get a different `dump_path`?

Comment (jack-h) [reply ovro_alert/buffer_trigger.py:10]: you do need to send a trigger to each (though I suppose this could be changed so that all pipelines listen to the same etcd key). Once all the nodes are properly set up, I would imagine half the pipelines should dump to `/data0` and the other half to `/data1` (such that the data path respects the NUMA affiliation of the pipeline)

**Comment by nkosogor** [comment]
@jack-h, can you please look at this part below? Is it right that if I want to use the highest frequencies I should trigger the pipelines with IDs 2 and 3? 
https://github.com/ovrocaltech/ovro-alert/blob/25ec16f52a7a2eaf9048d37d03850e71403bb03e/ovro_alert/lwa_alert_client.py#L9
And also I'm sending the data to /data0 and /data1 with corresponding IDs. Is it correct?
https://github.com/ovrocaltech/ovro-alert/blob/25ec16f52a7a2eaf9048d37d03850e71403bb03e/ovro_alert/lwa_alert_client.py#L28-L35

**Comment by jack-h** [comment]
TL;DR -- yes, I think you are correct on both counts

For my info more than yours, probably...

Mapping from the interleaved config file:

```
# lwa-gpu01
    10.41.0.65-10000: [ 560,  656]
    10.41.0.18-10000: [ 656,  752]
    10.41.0.65-20000: [2096, 2192]
    10.41.0.18-20000: [2192, 2288]
    # lwa-gpu02
    10.41.0.66-10000: [ 752, 848]
    10.41.0.17-10000: [ 848, 944]
    10.41.0.66-20000: [2288, 2384]
    10.41.0.17-20000: [2384, 2480]
...
```

Higher freq channels go to the X-engine NICs on port 20000.

From https://github.com/realtimeradio/caltech-bifrost-dsp/blob/a9cda5bcca08d2e41d54efa931e078a20134c65e/pipeline-control/lwa352_pipeline_control/lwa352_xeng_etcd_client.py#L18 these are indeed the last two pipelines. (IDs `2`, `3` of `0,1,2,3`).

Assuming (as is the intended setup) that `/data0` is associated with the NUMA node including CPU0, and `/data1` with the NUMA node including CPU 16, then pipeline 2 should write to `data0` and pipeline 3 to `data1`. (Based on https://github.com/realtimeradio/caltech-bifrost-dsp/blob/a9cda5bcca08d2e41d54efa931e078a20134c65e/pipeline-control/lwa352_pipeline_control/lwa352_xeng_etcd_client.py#L22)

**Review by caseyjlaw — APPROVED** [review]
